features/facebookImport/src/errors/polyIn-errors.js

```diff
@@ -0,0 +1,23 @@
+export class FileImportError extends Error {
```

Comment:
This is essentially "Failed to ${x} file". Maybe a single class?

Reply:
(It's likely this is from errorPopUp...)

Reply:
We just moved them in a file

Reply:
Right. Still, maybe a refactoring to DRY would be convenient (if you consider it so)